script/deploy-dev.mk

```diff
@@ -34,7 +34,10 @@ deploy-dev: deploy-dex deploy-openldap update-apiserver-etc-hosts
 		--values ./docs/user/manifests/kubeapps-local-dev-values.yaml \
 		--values ./docs/user/manifests/kubeapps-local-dev-auth-proxy-values.yaml \
 		--set useHelm3=true \
+		--set allowNamespaceDiscovery=true \
 		--set postgresql.enabled=true \
+		--set postgresql.volumePermissions.enabled=false \
+		--set postgresql.volumePermissions.image.pullPolicy=IfNotPresent \
```

Comment:
curious, why this?

Reply:
Woops. It was because I was having issues with postgres taking too long to start up when working on a slower connection (it's been trickier running locally rather than on dockerdev). So I was ensuring I could use an image I'd already loaded on the cluster. Removed.